jib-gradle-plugin/src/integration-test/resources/projects/broken_user/src/main/java/com/test/Empty.java

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
We can probably just reuse the existing `empty` test project and test multiple different build configurations, like we do under the `simple` project with `build.gradle` and `complex-build.gradle`.

Reply:
(We can do that with maven as well)

Reply:
Right, but I think we can just create a new build.gradle (called `brokenuser-build.gradle` or something), and pass the name of that into the `TestProject#build` call with the "-b" flag. This way we can make one build that succeeds with the correct parameters and a build the fails with different parameters, without having to duplicate this Empty.java.

Reply:
Ah, yeah, we can have another build file.